Runtime/Exporters/Exporter.cs

```diff
@@ -37,7 +37,7 @@ public virtual List<ValidationResult> CanExport()
 
         public virtual DirectoryInfo CreateExportDirectoryInfo()
         {
-            return Directory.CreateDirectory(Path.Combine(Map.exportSettings.OutputDirectory));
+            return Directory.CreateDirectory(Path.Combine(Map.projectFolder, ".treasured", Map.projectFolder));
```

Comment:
Should be using Map.exportSettings.OutputDirectory to make it backward compatible